tests/powder/correction_test.py

```diff
@@ -315,3 +319,43 @@ def test_apply_lorentz_correction_needs_coords():
     )
     with pytest.raises(KeyError):
         apply_lorentz_correction(da)
+
+
+def test_pdf_structure_factor_needs_q_coord():
+    da = sc.DataArray(sc.ones(sizes={'Q': 3}))
+    r = sc.array(dims='r', values=[2, 3, 4, 5.0])
+    with pytest.raises(KeyError):
+        compute_pdf_from_structure_factor(
+            da, sc.scalar(1.0, unit='1/angstrom'), r, use_filter=False
+        )
+
+
+@pytest.mark.parametrize('use_filter', [(False,), (True,)])
```

Comment:
```suggestion
@pytest.mark.parametrize('use_filter', [False, True])
```

Reply:
I think that didn't work for some reason. I'll try it again.